schemas/context/webpage.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/webpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Web Page",
+  "type": "object",
+  "description":
+    "Details about the web page where a web interaction occurred, as recorded by an `ExperienceEvent`.",
+  "definitions": {
+    "webpage": {
+      "properties": {
+        "xdm:siteSection": {
+          "title": "Site Section",
+          "type": "string",
+          "description":
+            "The normative name of the site section where this web page resides, which may be used to classify or categorize the interaction"
+        },
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description":
+            "The normative name of the web page. This name is not necessarily the page title or directy associate with page content, but is used to organize a site's pages for classification purposes"
+        },
+        "xdm:URL": {
+          "title": "URL",
+          "type": "string",
+          "format": "uri",
+          "description":
+            "The normative or usual URL of the web page.  This may or may not be the actual URL used to reach the page, which would be recorded using `WebLink`."
+        },
+        "xdm:server": {
```

Comment:
I think the name `server` is a clear misnomer here, when we are talking about a hostname.

Reply:
No, this is the actual host name that served the root html page. It is used for debugging purposes mostly.

Reply:
Actually thinking about it a bit, this variable is used in two different ways that are incompatible. It is used to hold the actual name of the server (hostname) and in other cases can be the domain. We cannot have this going forward. 
The domain should be extracted from the URL parameter

Reply:
I think having a `hostname` parameter that is able to override the hostname in the URL is sensible. e.g. you might want to treat traffic from `www-1.adobe.com` as if it originated from `www.adobe.com`. The point that I'm trying to make is that all of this is hostname arithmetics, not server arithmetics, because the traffic can come from the same or different servers (machines).